save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FossGraphIntroductionComponent.kt

```diff
@@ -0,0 +1,62 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.components.basic
+
+import csstype.ClassName
+import react.ChildrenBuilder
+import react.dom.html.ReactHTML
+
+private const val SAVE_FOSS_GRAPH_INTRO_MD = """
+    |## save-foss-graph
+    |It is a list of publicly disclosed information security vulnerabilities and exposures.
+"""
+
+private const val SAVE_FOSS_GRAPH_ADD_NEW_MD = """
+    |### New vulnerability
+    |You can suggest your own new vulnerability if you didn't find the required one in our list.
+    |After our approval, it will appear in the database.
+"""
+
+private const val SAVE_FOSS_GRAPH_HOW_TO_MD = """
+    |### How to add vulnerability in project?
+    |1. Create saveourtool [organization](#/create-organization) and [project](#/create-project);
+    |2. Go to your project's security tab;
+    |3. Create new problem and add vulnerability number;
+    |
+    |In case of any error feel free to [contact us](https://github.com/saveourtool/save-cloud/issues/new).
+"""
+
+@Suppress(
+    "MISSING_KDOC_TOP_LEVEL",
+    "MISSING_KDOC_ON_FUNCTION",
+)
+fun ChildrenBuilder.fossGraphIntroductionComponent() {
+    ReactHTML.div {
+        className = ClassName("card flex-md-column mb-1 box-shadow")
+        ReactHTML.div {
+            className = ClassName("card-body d-flex align-items-start")
+            ReactHTML.div {
+                ReactHTML.strong {
+                    className = ClassName("d-inline-block mb-2 text-info")
+                    +"Introducing"
+                }
+                markdown(SAVE_FOSS_GRAPH_INTRO_MD.trimMargin())
+            }
+            ReactHTML.div {
```

Comment:
```suggestion
            div {
```
and so on

copy-paste kills

Reply:
Fixed